src/isa/riscv64/local-include/intr.h

```diff
@@ -19,7 +19,30 @@
 
 #include <cpu/decode.h>
 #include "csr.h"
-
+#ifdef CONFIG_RVH
+enum {
+  EX_IAM, // instruction address misaligned
+  EX_IAF, // instruction address fault
+  EX_II,  // illegal instruction
+  EX_BP,  // breakpoint
+  EX_LAM, // load address misaligned
+  EX_LAF, // load address fault
+  EX_SAM, // store/amo address misaligned
+  EX_SAF, // store/amo address fault
+  EX_ECU, // ecall from U-mode or VU-mode
+  EX_ECS, // ecall from HS-mode 
+  EX_ECVS,// ecall from VS-mode
```

Comment:
Just use one "cause" enum. They are compatiable.
May add assert at the exception-handling method.